services/types.go

```diff
@@ -173,7 +173,7 @@ func (t *transaction) UnmarshalJSON(data []byte) error {
 		return err
 	}
 
-	input, err := hexutil.Decode(tw.Input)
+	data_, err := hexutil.Decode(tw.Data)
```

Comment:
The variable name is already taken - not sure what an appropriate alternative might be?